src/Manager.cc

```diff
@@ -50,18 +50,18 @@
 #include <utility>
 #include <vector>
 
-#include <ignition/common/Console.hh>
-#include <ignition/common/SignalHandler.hh>
-#include <ignition/common/SystemPaths.hh>
-#include <ignition/math/Rand.hh>
-#include <ignition/plugin/Loader.hh>
+#include <gz/common/Console.hh>
+#include <gz/common/SignalHandler.hh>
+#include <gz/common/SystemPaths.hh>
+#include <gz/math/Rand.hh>
+#include <gz/plugin/Loader.hh>
 
-#include "ignition/launch/config.hh"
-#include "ignition/launch/Plugin.hh"
+#include "gz/launch/config.hh"
```

Comment:
I think this may fail to compile unless we change this line

https://github.com/gazebosim/gz-launch/blob/8c78b6c937943ba1650e887c3a82dbfaea6c154f/src/Manager.cc#L1086